packages/apps/crm/src/hooks/use-crm-header-actions.ts

```diff
@@ -33,6 +39,28 @@ const useClientDetailsActions = () => {
   }, [mobileSettings, client]);
 };
 
+const useCatalogDetailsActions = () => {
+  const Colors = useThemeColor();
+  const navigation = useNavigation();
+  const I18n = useTranslator();
+
+  useEffect(() => {
+    headerActionsProvider.registerModel('crm_catalog_list', {
+      actions: [
+        {
+          key: 'newCatalog',
+          order: 10,
+          iconName: 'plus',
+          title: I18n.t('Crm_NewCatalog'),
+          iconColor: Colors.primaryColor.background,
+          onPress: () => navigation.navigate('CatalogFormScreen', {}),
+          showInHeader: true,
```

Comment:
Please add hideIf props to hide header action when catalog are not enabled on the web instance